components/automate-gateway/api/nodes/nodes.proto

```diff
@@ -124,6 +184,12 @@ service NodesService {
 			action: "infra:nodes:delete"
 		};
 	};
+
+	/*
+	Bulk create nodes
+
+	Creates multiple nodes given node data.
```

Comment:
```suggestion
	Creates multiple nodes from a list of node data.
```
hash?

Reply:
Does each node require  of a FQDN or IP address, a user-specified name, and a ssh or winrm credential reference?  Is there a shortcut? For example, could I provide a list of ip addresses and names, but only one credential reference?

Reply:
there are some shortcuts, i can't quite remember all the details but ill check